src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all 0.3s ease-in-out;
```

Comment:
don't use `all` with transition, you should specify every property

Reply:
Fixed.